index.html

```diff
@@ -1,43 +1,80 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="utf-8" />
-        <title>My personal website</title>
-    </head>
-    <body>
-        <header></header>
-        <h1>Welcome to my personal website</h1> 
-        <nav>
-            <ul>
-                <li>Home</li>
-                <li>
-                    <a href="About Me">About Me</a>
-                </li>
-                <li>Contact</li>
-                <li>Career</li>
-            </ul>
-            </nav>
-        </header>
-        <main>
-            <section>
-                <h2>About me</h2>
-                <img
-                src="/images/Passbild_Alexander_Schulte.jpg" alt="Alex profile picture">
-                <br>
-                <a href="https://github.com/Alexander-Schulte">ty my GitHub profil</a>
-                <p>
-                    elrmvlekv lkfmvlkmrwklrmvrlkmvkrmvwkrv
-                </p>
-            </section>
-            <section>
-                <h2>Things I like to do</h2>
-                <ul>
-                    <li>Football</li>
-                    <li>Cooking</li>
-                    <li>Running</li>
-                </ul>
-            </section>
-        </main>
-        <footer>made with web technology</footer>
-    </body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="utf-8" />
+    <title>Alexander Schulte</title>
+  </head>
+  <body>
+    <header>
+      <h1>Welcome to the website of Alexander Schulte</h1>
+      <nav>
+        <ul>
+          <li>Home</li>
+          <li>Contact</li>
+          <li>
+            <a href="#About Me">About Me</a>
+          </li>
+          <li>
+            <a href="#Career">Career</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section>
+        <h2>This is me!</h2>
+        <img
+          width="300"
+          height="300"
+          src="/images/AlexanderSchulte_Brighton.JPG"
+          alt="Alexander Schulte profile picture"
+        />
+
+        <h2 id="About Me">About Me</h2>
```

Comment:
This seems to be odd. Would recommend this:
```suggestion
        <h3 id="About Me">About Me</h3>
```
The headings present level of sections headings. h1 is the highest, h6 the lowest. You used the first h2 in line 25 for the "This is me!". I would assume that the "about me" line is something under the "This is me". I hope you understand what I mean ;-) See more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements).